src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -140,6 +140,11 @@ public override void ExecuteCommand()
             {
                 packArgs.SymbolPackageFormat = PackArgs.GetSymbolPackageFormat(SymbolPackageFormat);
             }
+            else if (Symbols)
+            {
+                packArgs.Logger.Log(LogMessage.CreateWarning(NuGetLogCode.NU5132, LocalizedResourceManager.GetString("PackageCommandDefaultSymbolPackageFormatChanging")));
```

Comment:
Do we want to warn in nuget.exe? 

I'm worried about us suggesting that people use snupkg, while their pdbs are not portable.